ReSharper.FSharp/src/FSharp.Psi.Features/src/Parsing/FSharpTreeBuilderBase.fs

```diff
@@ -678,6 +678,11 @@ type FSharpTreeBuilderBase(lexer, document: IDocument, lifetime, path: VirtualFi
 
         let rec loop paramGroups returnInfo synType =
             match paramGroups, synType with
+            | _, SynType.WithGlobalConstraints(synType, constraints, range) ->
```

Comment:
Is the loop over parameter groups an appropriate place to add this change to? Can these constraints go after each parameter group?

Reply:
no, only at the end of the signature, possibly combined via 'and'